### DISCUSSION TITLE
Problemas Safe Exam Browser 2.4.1 y win7 7601

### DISCUSSION THREAD

**Discussion by reyesruizrodrigo** [SafeExamBrowser/seb-win-refactoring#215] [Q&A]
Estimados al momento de ejecutar SEB en al versión de windows7 la barra inferior donde se muestran las herramientas para cerrar la aplicación no se muestra.

![boton](https://user-images.githubusercontent.com/71410017/135653524-de282858-ab26-434d-8b0e-49813fcd36a2.png)

Se realizo la configuración de las contraseñas para salir de la aplicación con SEB Config Tool y al momento de cerrar SEB el computador muestra pantalla negra solo permitiéndome mover el cursor del mouse pero no termina SEB completamente, necesitando reiniciar manualmente el equipo para solucionar el inconveniente.

Agradezco su ayuda para solucionar el problema.

Dear at the moment of executing SEB in the windows7 version, the bottom bar where the tools for the application are shown is not shown.

![boton](https://user-images.githubusercontent.com/71410017/135653524-de282858-ab26-434d-8b0e-49813fcd36a2.png)

The password settings were made to exit the application with SEB Config Tool and when closing SEB the computer shows a black screen only allowing me to move the mouse cursor but it does not finish SEB completely, needing to manually restart the equipment to solve the problem.

I appreciate your help in solving the problem.

**Comment by dbuechel** [comment]
La versión mínima del sistema operativo para usar SEB 3.x es Windows 8.1. Windows 7 no mas esta apto para usar con la ultima versión de SEB. Ademas le recomiendo antes de todo para razones de seguridad actualizar su sistema operativo a Windows 10 o 11 lo antes posible.

The minimal operating system version for SEB 3.x is Windows 8.1. Windows 7 isn't supported anymore for the latest release version of SEB. I furthermore recommend you first and foremost for security reasons to update your operating system to Windows 10 or 11 as soon as possible.
